### PR TITLE
change go version

### PR DESCRIPTION
**Pull request by madharis9**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,5 @@
-module github.com/kikan/fsm
+module lunox
```
Comment (yudhiana99) [inline go.mod:1]: change to 'github.com/mataharibiz/lunox'

**Review by yudhiana99 — COMMENTED** [review]
please check update

File: go.mod (hunk 1 above)
Comment (madharis9) [reply go.mod]: updated mas @yudhiana99 , please re-review, thank you

**Review by yudhiana99 — APPROVED** [review]
ok , LGTM

**Comment by iqbalprawira78** [comment]
LGTM

**Pull request merged**